exchanges/order/order_test.go

```diff
@@ -1076,72 +898,60 @@ func TestUpdateOrderFromModifyResponse(t *testing.T) {
 	updated := time.Now()
 
 	pair, err := currency.NewPairFromString("BTCUSD")
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
 
 	om := ModifyResponse{
-		ImmediateOrCancel: true,
-		PostOnly:          true,
-		Price:             1,
-		Amount:            1,
-		TriggerPrice:      1,
-		RemainingAmount:   1,
-		Exchange:          "1",
-		Type:              1,
-		Side:              1,
-		Status:            1,
-		AssetType:         1,
-		LastUpdated:       updated,
-		Pair:              pair,
+		TimeInForce:     PostOnly | GoodTillTime,
+		Price:           1,
+		Amount:          1,
+		TriggerPrice:    1,
+		RemainingAmount: 1,
+		Exchange:        "1",
+		Type:            1,
+		Side:            1,
+		Status:          1,
+		AssetType:       1,
+		LastUpdated:     updated,
+		Pair:            pair,
 	}
 
 	od.UpdateOrderFromModifyResponse(&om)
+	assert.NotEqual(t, UnknownTIF, od.TimeInForce)
+	assert.True(t, od.TimeInForce.Is(GoodTillTime))
+	assert.True(t, od.TimeInForce.Is(PostOnly))
+	assert.Equal(t, 1., od.Price)
+	assert.Equal(t, 1., od.Amount)
+	assert.Equal(t, 1., od.TriggerPrice)
+	assert.Equal(t, 1., od.RemainingAmount)
+	assert.Empty(t, od.Exchange, "Should not be able to update exchange via modify")
+	assert.Equal(t, "1", od.OrderID)
+	assert.Equal(t, Type(1), od.Type)
+	assert.Equal(t, Side(1), od.Side)
+	assert.Equal(t, Status(1), od.Status)
+	assert.Equal(t, asset.Item(1), od.AssetType)
+	assert.Equal(t, od.LastUpdated, updated)
+	assert.Equal(t, "BTCUSD", od.Pair.String())
+	assert.Nil(t, od.Trades)
+}
 
-	if !od.ImmediateOrCancel {
-		t.Error("Failed to update")
-	}
-	if !od.PostOnly {
-		t.Error("Failed to update")
-	}
-	if od.Price != 1 {
-		t.Error("Failed to update")
-	}
-	if od.Amount != 1 {
-		t.Error("Failed to update")
-	}
-	if od.TriggerPrice != 1 {
-		t.Error("Failed to update")
-	}
-	if od.RemainingAmount != 1 {
-		t.Error("Failed to update")
-	}
-	if od.Exchange != "" {
-		t.Error("Should not be able to update exchange via modify")
-	}
-	if od.OrderID != "1" {
-		t.Error("Failed to update")
-	}
-	if od.Type != 1 {
-		t.Error("Failed to update")
-	}
-	if od.Side != 1 {
-		t.Error("Failed to update")
-	}
-	if od.Status != 1 {
-		t.Error("Failed to update")
-	}
-	if od.AssetType != 1 {
-		t.Error("Failed to update")
-	}
-	if od.LastUpdated != updated {
-		t.Error("Failed to update")
-	}
-	if od.Pair.String() != "BTCUSD" {
-		t.Error("Failed to update")
-	}
-	if od.Trades != nil {
-		t.Error("Failed to update")
+func TestTimeInForceIs(t *testing.T) {
```

Comment:
If you could also put all the TimeInForce tests in the same area too, thank you

Reply:
suggestion: Can have its own go file `timeinforce.go`